.gitignore

```diff
@@ -6,14 +6,10 @@
 /build
 /dist
 /MANIFEST
-/setuptools-3.1-py2.7.egg
-/setuptools-3.1.zip
 /stone.egg-info
 .arcconfig
 .DS_Store
 .idea
-parser.out
```

Comment:
Is that something that a Stone developer might create? Should we be automating tests or other checks that invoke those things?